### PR TITLE
1st part of the code challenge

### PR DESCRIPTION
**Pull request by kaaresylow**

### CODE REVIEW

**Comment by nealstewart** [comment]
You've committed your .idea files. We don't commit those locally, so I'd:

1. Remove them and commit the removal.
2. Add .idea/* to your .gitignore

**Comment by nealstewart** [comment]
You've also committed .DS_STORE files. Do the same with those.

File: test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+test
```
Comment (nealstewart) [inline test.txt]: Remove this file as well.

As general practice, I would recommend adding files one by one when creating git commits.

I generally add changes line by line. Are you using a Git UI of some kind, or just the command line?

**Comment by kaaresylow** [comment]
Thanks. I'm using git Tower and hit the stage all button a little too fast.
I have updated the gitignore file and stopped the tracking of the unnecessary files.

**Comment by nealstewart** [comment]
@kaaresylow Looks good, other than the git commit history.

Please rebase into a single commit.

`git fetch; git rebase -i origin/master`

File: web/index.jsp [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<%--
```
Comment (nealstewart) [inline web/index.jsp]: Don't commit this sort of auto-generated documentation in future.

It's like leaving the tag on a new shirt.

Comment (kaaresylow) [inline web/index.jsp]: Sorry, I'm just trying to figure out how git rebase works and screwed it up a bit.

**Comment by nealstewart** [comment]
One more rebase and we're good! ;)

**Comment by nealstewart** [comment]
Lovely :+1:

**Pull request merged**